skorch/tests/test_regressor.py

```diff
@@ -36,6 +36,23 @@ def net(self, net_cls, module_cls):
             lr=0.1,
         )
 
+    @pytest.fixture(scope='module')
+    def multi_target_data(self, multi_target_regression_data):
```

Comment:
Since this fixture doesn't really do anything, you can remove it. Below, in the test function that uses the data, you can directly reference the fixture you defined in `conftest.py`.